index.html

```diff
@@ -0,0 +1,109 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <nav>
+
+    </nav>
+  </header>
+  <section>
+    <div class="title-decoration">
+      <img class="title-image" src="triangle.svg" alt="Front-End Tools"/>
+      <p class="title-text">Front-End Tools</p>
+    </div>
+    <div class="container-tech">
+      <div class="tech">
+          <img  class="tech-img" src="react_icon.svg" alt="ReactJS">
+          <p class="text-big">ReactJS</p>
+          <br>
```

Comment:
`<br>` tags shouldn't be used to make gaps between elements, instead adjust the margin styled by CSS